lib/ExpandingTextArea/README.md

```diff
@@ -17,7 +17,6 @@ A component that renders a textarea that automatically resizes depending on its
 | className           | String        | ''        | No       | Sets the component className                                                           |
 | minRows             | Number        | 1         | No       | Sets the minimum amount of rows for the textArea                                       |
 | maxRows             | Number        | 1         | No       | Sets the maximum amount of rows for the textArea                                       |
-| widthMedium             | Boolean        | false         | No       | Sets a fixed medium width                                       |
```

Comment:
is this just not a thing?

Reply:
Yeah it's a prop I added, then removed but didn't remove the docs!